packages/govern-console/src/lib/web3-utils.js

```diff
@@ -95,59 +120,37 @@ export function shortenAddress(address, charsLength = 4) {
   )
 }
 
-export function getNetworkType(chainId = env('CHAIN_ID')) {
-  chainId = String(chainId)
-
-  if (chainId === '1') return 'main'
-  if (chainId === '3') return 'ropsten'
-  if (chainId === '4') return 'rinkeby'
-
-  return DEFAULT_LOCAL_CHAIN
-}
-
-export function getNetworkNode(chainId = env('CHAIN_ID')) {
-  chainId = String(chainId)
-
-  if (chainId === '1') return 'https://mainnet.eth.aragon.network/'
-  if (chainId === '4') return 'https://rinkeby.eth.aragon.network/'
-}
-
-export function sanitizeNetworkType(networkType) {
-  if (networkType === 'private') {
-    return 'localhost'
-  } else if (networkType === 'main') {
-    return 'mainnet'
-  }
+export function sanitizeNetworkType(networkType: string): string {
+  if (networkType === 'private') return 'localhost'
+  if (networkType === 'main') return 'mainnet'
   return networkType
 }
 
-export function isLocalOrUnknownNetwork(chainId = env('CHAIN_ID')) {
+export function isLocalOrUnknownNetwork(chainId = env('CHAIN_ID')): boolean {
   return getNetworkType(chainId) === DEFAULT_LOCAL_CHAIN
 }
 
-export function hexToAscii(hexx) {
-  const hex = hexx.toString()
-  let str = ''
-  for (let i = 0; i < hex.length && hex.substr(i, 2) !== '00'; i += 2) {
-    str += String.fromCharCode(parseInt(hex.substr(i, 2), 16))
-  }
-  return str
-}
```

Comment:
I couldn’t find any use of this function, also I am not sure what `hexx` is supposed to be, other than a `string`? :thinking: 

Let me know if it should stay!

Reply:
And actually, now that we have typings, we could probably move this file and `web3-contracts` to use the `.ts` right?

Reply:
Oh I completely missed the fact that it wasn’t a `.ts` file! Will do :+1:

Reply:
Done in https://github.com/aragon/govern/pull/261/commits/474834e3b35bb75b3567b2fa45b7723adff36cc1 − that required a few new changes :-)